### PR TITLE
PYIC-2185 Update unhappy path content

### PR DESCRIPTION
**Pull request by wilsond-gds**
## Proposed changes

This PR standardises unhappy path content in core; it is related to other content changes for Auth, Address and KBVs

### What changed

The .json files and related nunjucks files have updated content and keys.

### Why did it change

Changes were requested by the content designer.

### Issue tracking

- [PYIC-2185](https://govukverify.atlassian.net/browse/PYIC-2185)

[PYIC-2185]: https://govukverify.atlassian.net/browse/PYIC-2185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/views/errors/page-not-found.njk [hunk 1]
```diff
@@ -5,9 +5,12 @@
 
 {% block content %}
   <h1 class="govuk-heading-l">{{'error.pageNotFound.header' | translate }}</h1>
-  <p class="govuk-body">{{'error.pageNotFound.content.paragraph1' | translate }}</p>
-  <p class="govuk-body">{{'error.pageNotFound.content.paragraph2' | translate }}</p>
-  <p class="govuk-body">{{'error.pageNotFound.content.paragraph3' | translate }}</p>
+    <p class="govuk-body">{{'error.pageNotFound.content.paragraph1' | translate }}</p>
```
Comment (AmritSidhu) [inline src/views/errors/page-not-found.njk]: `<p>` tags dont need to be indented, should be at the same level at the `<h1>`

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -75,9 +75,11 @@
       "title": "Page not found",
       "header": "Page not found",
       "content": {
-        "paragraph1": "If you typed the web address, check it’s correct.",
-        "paragraph2": "If you pasted the web address, check you copied the entire address.",
-        "paragraph3": "You can also go back to the service you were trying to use and start again. You can look for the service using your search engine or find it from the GOV.UK homepage."
+        "paragraph1": "We cannot find the page you're looking for.",
```
Comment (AmritSidhu) [inline src/locales/en/translation.json]: Different apostrophes`'` - `’`

File: src/views/ipv/page-ipv-success.njk [hunk 3]
```diff
@@ -4,9 +4,11 @@
 {% set googleTagManagerPageId = "pageIpvSuccess" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l" id="header" data-page="{{googleTagManagerPageId}}">{{'pages.pageIpvSuccess.header' | translate }}</h1>
-  <p class="govuk-body">{{'pages.pageIpvSuccess.content.paragraph1' | translate | safe }}</p>
-  <p class="govuk-body">{{'pages.pageIpvSuccess.content.paragraph2' | translate }}</p>
 
-  {% include 'shared/journey-next-form.njk' %}
+    <h1 class="govuk-heading-l" id="header"
```
Comment (AmritSidhu) [inline src/views/ipv/page-ipv-success.njk]: Check indentation here to and make sure is consistent with the other pages

**Review by AmritSidhu — CHANGES_REQUESTED** [review]

File: src/locales/en/translation.json (hunk 2 above)
Comment (wilsond-gds) [reply src/locales/en/translation.json]: 😱 how did I miss that

File: src/views/ipv/page-ipv-success.njk (hunk 3 above)
Comment (wilsond-gds) [reply src/views/ipv/page-ipv-success.njk]: Found a new setting in WebStorm - should be 2 spaces now with all the others

File: src/views/errors/page-not-found.njk (hunk 1 above)
Comment (wilsond-gds) [reply src/views/errors/page-not-found.njk]: ah yes, fixed.

**Review by AmritSidhu — APPROVED** [review]

**Pull request merged**